test/lib/rails-jquery-autocomplete_test.rb

```diff
@@ -3,7 +3,7 @@
 module RailsJQueryAutocomplete
   class RailsJQueryAutocompleteTest < ActionController::TestCase
     ActorsController = Class.new(ActionController::Base)
-    ActorsController.autocomplete(:movie, :name)
+    ActorsController.autocomplete(:name, { :movie => :name })
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.